docs/user-guide/quick-start/admin-gui.md

```diff
@@ -2,81 +2,215 @@
+<figure markdown="span">
+![PermissionsView](quick-start-images/10PermissionsView.png)
+<figcaption>Hover over the permissions to view their description. For example, hovering over <strong>Manage groups</strong> shows that the permission allows the user: Access and manage of all groups</figcaption>
+</figure>
 
-![New Group Button](quick-start-images/new-group-button.png)
+### Edit User status
 
-* Choose a name for the group and click **Create**.
+As an administrator, you have the ability to manage users' permissions, statuses (active or inactive), and passwords.
 
-![New Group Create Button](quick-start-images/create-group-button.png)
+To edit a user's status, follow these steps:
 
-### Edit and Add Members to the Groups
+1. **Edit User Status:**
+    * To change a user's status to active or inactive, click on the three dots on the left side of the user's name.
+    * Select **Deactivate** to turn the user inactive immediately. To reactivate a deactivated user, select **Activate**.
 
-* Add new members to your group by selecting from the list of users or using the search bar to find specific members. 
-* Click Add member to add the selected user to the group.
+    <figure markdown="span">
+    ![DeactivateUsers](quick-start-images/11DeactivateUsers.png)
+    <figcaption>Edit user status. To activate or deactivate a user, click on the three dots next to the user’s name and select either **Activate** or **Deactivate**. This change will take effect immediately</figcaption>
+    </figure>
 
-![Create New Member](quick-start-images/add-new-member.png)
+2. **Change User Password:**
+    * To change a user's password, click on the three dots next to the user's name.
+    * Select **Change Password**. A new window will appear, prompting you to authenticate yourself as an administrator.
+    * After authenticating, enter and confirm the new password for the selected user. Ensure that the new password meets the required security criteria.
+    * A confirmation window will notify you that the password has been successfully changed.
+   
+    <figure markdown="span">
+    ![ChangePasswords](quick-start-images/12ChangePasswords.png)
+    <figcaption>Change a user’s password. Click on the menu next to the user's name, authenticate yourself as an admin (e.g. Genestack Superuser), and then set a new password for the user (e.g. New User 2024). Ensure the new password complies with security guidelines</figcaption>
+    </figure>
 
-### Delete Groups
-!!! warning "To add, update, and delete groups "Manage groups" permission is required."
+## Add, update, and delete Groups
+
+#### **Creating a Group**
+
+To create a new group manually, follow these steps:
+
+1. **Access the Groups Section**
+   * Click on the three-line menu button at the top left of the dashboard to access the **Groups** section.
+
+    !!! info "Important"
+        In order to be able to create, edit and delete groups, the user should have the **Manage organization** permission.
```

Comment:
same here, can we please somehow mention that not in all cases the user should have this permission? or at least mention that we are talking about managing any group that even have been created by some other user?